pyanaconda/core/regexes.py

```diff
@@ -194,3 +194,6 @@
 
 # Octet of MAC address
 MAC_OCTET = re.compile(r'[a-fA-F0-9][a-fA-F0-9]')
+
+# Name of initramfs connection created by NM based on MAC
+NM_MAC_INITRAMFS_CONNECTION = re.compile(r'^([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$')
```

Comment:
What is the `[:]` part supposed to do?

Reply:
Well, I think there could be also just ':' - it is a MAC octet separator.

Reply:
I've updated the patch '[:]' -> ':'

Reply:
We really should have this abstracted somewhere. This is really bad.

Reply:
Well, this *is* the abstraction, isn't it...